plugins/indexer/nexus-indexer-lucene-plugin/src/main/resources/static/js/Sonatype/repoServer/RepositoryIndexBrowserContainer.js

```diff
@@ -145,6 +145,8 @@ define('Sonatype/repoServer/RepositoryIndexBrowserContainer', function() {
 
                       var resourceURI = Sonatype.config.servicePath + '/repositories/' + options.cbPassThru.node.attributes.repositoryId + '/content' + json.data.repositoryPath;
 
+                      var payload = options.cbPassThru.container.payload;
```

Comment:
Yes, when you construct the repository browser you pass payload information to it.
`var payload = (options.cbPassThru.container.payload) ?  options.cbPassThru.container.payload : options.cbPassThru.container.repositoryBrowser.payload;`

That way I believe you can avoid passing a null format (not that it is used by the gav search).